pkgs/applications/radio/sdrplay/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchurl, autoPatchelfHook, libudev }:
+let
+  arch = if stdenv.isx86_64  then "x86_64"
+    else if stdenv.isi686    then "i686"
+    else throw "unsupported architecture";
+in
+stdenv.mkDerivation rec {
+  name = "sdrplay";
+  version = "3.07.1";
+
+  # soapysdrplay also needs to full so name
+  soName = "libsdrplay_api.so." + lib.concatStringsSep "." (lib.take 2 (builtins.splitVersion version));
+
+  src = fetchurl {
+    url = "https://www.sdrplay.com/software/SDRplay_RSP_API-Linux-${version}.run";
+    sha256 = "1a25c7rsdkcjxr7ffvx2lwj7fxdbslg9qhr8ghaq1r53rcrqgzmf";
+  };
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [ libudev stdenv.cc.cc.lib ];
+
+  unpackPhase = ''
+    sh $src --noexec --target source
+  '';
+
+  sourceRoot = "source";
+
+  buildPhase = "";
+
+  installPhase = ''
+    mkdir -p $out/{bin,lib,include,lib/udev/rules.d}
+    cp ${arch}/${soName} $out/lib/
+    cp ${arch}/sdrplay_apiService $out/bin/
+    cp -r inc/* $out/include/
+    cp 66-mirics.rules $out/lib/udev/rules.d/
+  '';
+
+  meta = with lib; {
+    description = "SDRPlay API";
+    homepage = "https://www.sdrplay.com/downloads/";
+    license = licenses.unfree;
```

Comment:
Is there a link to the license agreement/conditions that we could add here?

Reply:
I didn't find any direct license page / link, but a [documentation PDF](https://www.sdrplay.com/docs/SDRplay_API_Specification_v3.07.pdf). I've now included that in in the `longDescription`. Is that the right place for it?